src/templates/components/newsStoryTeaser/index.tsx

```diff
@@ -11,7 +11,7 @@ export const NewsStoryTeaser = ({
   introText,
 }: FormattedNewsStoryTeaser) => {
   const TitleTag = ({ children, className }) => {
-    const Heading = headingLevel ? headingLevel : 'h2'
+    const Heading = (headingLevel ? headingLevel : 'h2') as React.ElementType
```

Comment:
Apparently React 19 has more strict type checking for what we can use as a `<Component>`. 🤷